lib/puppet/provider/sensu_postgres_config/sensuctl.rb

```diff
@@ -5,6 +5,15 @@
 
   mk_resource_methods
 
+  def min_version
```

Comment:
Are there spec tests that show the warning when the version is below the minimum?

Reply:
Not possible it seems.

This is not catching the warning despite it clearly getting called:
```ruby
expect(Puppet).to receive(:warning)
```

Ran the code in Vagrant with Sensu Go 6.0.0 installed and get this:

```
Warning: Sensu_postgres_config[postgresql] Property strict skipped, does not meet minimum Sensu Go version of 6.1.0
Warning: Sensu_postgres_config[postgresql] Property batch_buffer skipped, does not meet minimum Sensu Go version of 6.1.0
Warning: Sensu_postgres_config[postgresql] Property batch_size skipped, does not meet minimum Sensu Go version of 6.1.0
Warning: Sensu_postgres_config[postgresql] Property batch_workers skipped, does not meet minimum Sensu Go version of 6.1.0
```

This must be either a limitation with rspec or some quirk with how Puppet.warning is defined in Ruby to make the call not something we can test.  The best we can do is test that the `next` line is called which is tested by showing the new properties are omitted from being set when not supported.